src/pythae/models/beta_vae/beta_vae_model.py

```diff
@@ -71,7 +71,7 @@ def forward(self, inputs: BaseDataset, **kwargs):
         loss, recon_loss, kld = self.loss_function(recon_x, x, mu, log_var, z)
 
         output = ModelOutput(
-            reconstruction_loss=recon_loss,
```

Comment:
Update this [test](https://github.com/ravih18/pythae/blob/a1c8cce2be8e94ecb7283e7f65e568581b82df14/tests/test_BetaVAE.py#L287-L289) in `test_BetaVAE` with `recon_loss` to avoid failure